src/components/Card/Card.jsx

```diff
@@ -1,9 +1,13 @@
-// import "./Card.scss";
+import React from "react";
+import "./Card.scss";
 
-// const Card = ({ title, url, image }) => {
-//   return (
+const Card = (cardInfo) => {
+  const { title, url, image } = cardInfo
+  return <div className="card">
+  <h2>{title}</h2>
+  <a href={url}>Book now!</a>
+  <img src={image} alt={title}></img>
+  </div>
+};
```

Comment:
Well done Hadika, you are doing well. But for more readability you can write your {title, url, image} in a parameter like, const Card = ({ title, url, image }) => {

Reply:
The reason why I wrote like this is because I am using props. This is how you are supposed to use props. 
If you are unclear or need any more explanation, then please let me know.